### DISCUSSION TITLE
Is it possible to add content information to responses?

### DISCUSSION THREAD

**Discussion by martin-thoma** [python-restx/flask-restx#589] [Q&A]
I currently have an endpoint with the decorator:

```python
@api.route("/action/<string:action_id>")
class ApiAction(Resource):
    @api.doc(
        responses={200: "The action object", 404: "Action not found"}
    )
    def get(self, action_id: UUID):
        ...
```

which generates:

```
      responses:
        "200":
          description: Newly created action
```

I want to generate:

```
      responses:
        "200":
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Action'
          description: Newly created action
```

Is it possible to add this content data with Flask-Restx

**Comment by peter-doggart** [comment]
You can specify a model that gets returned from flask-restx using something like `@api.response(200, 'Success', model)`. (You can have multiple responses per endpoint too). 

See this bit of the docs: https://flask-restx.readthedocs.io/en/latest/swagger.html#documenting-with-the-api-response-decorator
